functions/confirm_registration/main.py

```diff
@@ -35,6 +43,16 @@ def confirm_registration(request):
         ), 422
     request_data = request.get_json()
 
+    if not _is_language_valid(request_data):
+        return False, (jsonify(
+            {
+                "status": "failed",
+                "message": "Set lang parameter to pl or en"
+            }
+        ), 422)
```

Comment:
Czy wartość tego `message` zobaczymy w StackDriverze? Jeśli nie, to nie będziemy wiedzieć z jakiego powodu nie powiodło się żądanie... Widzę na StackDriver tylko `"Function execution took 119 ms, finished with status code: 422" `. Może warto byłoby zwracać różne HTTP Status codes dla różnych błędów?

Reply:
Dodałem logowanie. Inny kod nie jest potrzebny mobilkom, a na driverze będzie widać nasz log z apki.

Reply:
@kstrz w którym miejscu jest dodane to logowanie?

Reply:
W funkcji `_is_language_valid`
https://github.com/ProteGO-app/backend/pull/56/commits/f3b1eef936fe2e24ec75460dd437499a92f4ae3b